CMakeLists.txt

```diff
@@ -4,13 +4,14 @@
 #
 # Project-wide settings
 CMAKE_MINIMUM_REQUIRED(VERSION 3.1.0)
+ENABLE_LANGUAGE(C)
 
 # Name of the project.
 #
 # CMake files in this project can refer to the root source directory
 # as ${OpenMVS_SOURCE_DIR} and to the root binary directory as
 # ${OpenMVS_BINARY_DIR}.
-PROJECT(OpenMVS)
+PROJECT(OpenMVS LANGUAGES CXX CUDA)
```

Comment:
does it work if we change enlabe_language(C) to project(OpenMVS LANGUAGES C CXX CUDA)

Reply:
It works.